Sources/Perception/PerceptionRegistrar.swift

```diff
@@ -150,3 +151,44 @@ private func perceptionCheck() {
     }
   #endif
 }
+
+var isInSwiftUIStack: Bool {
+  for callStackSymbol in Thread.callStackSymbols {
+    guard
+      let symbol = callStackSymbol.split(separator: " ").dropFirst(3).first,
+      symbol.hasPrefix("$"),
+      String(symbol).demangled.hasPrefix("protocol witness for SwiftUI.View.body.getter : ")
```

Comment:
@mbrandonw I dunno if there's a better way to demangle, but this seems like a reliable way to detect if we're in a view body stack.

Reply:
Nice, looks good! Gonna merge.